src/main/java/mServer/tool/MserverDaten.java

```diff
@@ -31,13 +32,19 @@
     public static MserverListeSuchen listeSuchen = new MserverListeSuchen();
     public static MserverListeUpload listeUpload = new MserverListeUpload();
     public static boolean debug = false;
+    public static boolean restart = false;
```

Comment:
Ich habe mich der Einfachheit halber erstmal an den vorhandenen Stil gehalten, also genau wie bei "debug" und den Listen.
Gegen eine saubere Kapselung spricht natürlich nichts, nur wären dies dann größere Änderungen gewesen, die in dem Fall unabhängig bearbeitet werden sollten. Gerade bei dieser Klasse wäre das ein mit etwas größerem Aufwand auch Richtung testen verbunden, da diese Daten an vielen Stellen genutzt werden, wenn ich das richtig gesehen hatte.